great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -135,15 +134,6 @@ def _init_variables(self) -> FileDataContextVariables:
         )
         return variables
 
-    def add_store(self, store_name: str, store_config: dict) -> Optional[Store]:
```

Comment:
Any chance we can make the `add_store` on the `AbstractDC` never return `None`?

Reply:
yup modified the underlying helper that does this so we always return an obj